src/main.js

```diff
@@ -201,16 +202,14 @@ function addToArray(arr, item) {
 function savePoster() {
   if (!savedPosters.includes(currentPoster)) {
     savedPosters.push(currentPoster);
-    for (var i = 0; i < savedPosters.length; i++) {
-      var imageURL = savedPosters[i].imageURL;
-      var title = savedPosters[i].title;
-      var quote = savedPosters[i].quote;
-      var sectionOpen = `<section class='mini-poster'>`;
-      var imgElement = `<img src=${imageURL}>`;
-      var h2Element = `<h2>${title}</h2>`;
-      var h4Element = `<h4>${quote}</h4>`;
-      var sectionClose = `</section>`;
-      savedParent.innerHTML += sectionOpen + imgElement + h2Element + h4Element + sectionClose;
-    };
+    var imageURL = currentPoster.imageURL;
```

Comment:
Good job catching the unnecessary loop and replacing it with the properties of our currentPoster instance